contribs/gmf/less/iphone.less

```diff
@@ -0,0 +1,20 @@
+// Hack for ios Safari browser (UI overlapping on iOS 11)
+@media all and (orientation: landscape) {
+  @ios-margin: 5rem;
+
+  .ios-margin-top {
+    margin-top: @ios-margin;
+  }
+
+  .ios-zoom-btn {
+    top: (@app-margin + @map-tools-size + @app-margin) + @ios-margin !important;
```

Comment:
You shouldn't use !important...

Reply:
not working without it :-(

Reply:
your selector isn't enough specific

Reply:
My specific selectors are located in the javascript file (I need to do it as it's from the javascript that I activate or not the UI tweak). In the less I only set the properties I want to override with a simple name that I can retrieve easily in the javascript code.

The classes are added using $('selector').addClass('className').